python/README.md.in

```diff
@@ -55,6 +55,8 @@ CUDA-Q with all its dependencies:
 [//]: # (Begin conda install)
 
 ```console
+conda tos accept --override-channels --channel https://repo.anaconda.com/pkgs/main
```

Comment:
Looking at this, it is probably better to not have that in here after all. We should probably just put this in the respective scripts. You can search for miniconda in the codebase, and just run that after each installation.